wallet/README.md

```diff
@@ -1,14 +1,7 @@
 # Wallet
 
 ## Testing ERC Contracts
-For testing purposes, account `0x9C8B2276D490141Ae1440Da660E470E7C0349C63` with privateKey `0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e` has been setup in testnet wit some 
-Ether and some ERC tokens.
-The following is the list of test ERC contracts deployed:
-- 0xb5acbe9a0f1f8b98f3fc04471f7fe5d2c222cb44 => ERC20 contract
-- 0x103ac4b398bca487df8b27fd484549e33c234b0d => ERC721 contract. Existing token Ids are 1,2, and 3
-- 0x9635c600697587dd8e603120ed0e76cc3a9efe4c => ERC1155 contract. Existing token Ids are 1 through 4.
-
-When starting the wallet, you can add the address of these tokens if they do not appear by default.
+For testing purposes, account `0x9C8B2276D490141Ae1440Da660E470E7C0349C63` with privateKey `0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e` has been setup in testnet wit some Ether and some ERC tokens.
```

Comment:
We should never have private keys in our repo even if we were to use it in ropsten.
Bots can steal ropsten ether, creating trouble for us.
Can we remove this please?
As for code, we will pass through environment variables and the environment variable files can be local and added to `.gitignore`

Reply:
I agree, but this key is present in many places in the repo. Just do a search.

Reply:
That is true. 
I will create a new issue for having this removed instead of handling this here
The history in github will still hold this key even after removing this key, we might have to change this to new keys while tackling the issue and not push it. I will mention this in the issue too